docs/chapters/introduction/content.uk-UA.md

```diff
@@ -0,0 +1,10 @@
+# Короткий вступ
+
+Давайте розпочнемо з добрих новин: криві Безьє, про які ми говоритимемо, ви зможете побачити далі на графіках. Ці криві розпочинаються у якійсь певній точці, і закінчуються у якійсь певній точці.  Їх кривизна залежить від однієї або кількох "проміжних" контрольних точок. Зараз, оскільки всі графіки на цій сторінці інтерактивні, поекспериментуйте трохи з цими кривими. Клікніть на точку мишкою й потягніть - так ви зможете відчути, як форма кривої змінюється в залежності від ваших дій.  
+
+<div className="figure">
+  <Graphic inline={true} title="Квадратичні криві Безьє" setup={ this.drawQuadratic } draw={ this.drawCurve }/>
+  <Graphic inline={true} title="Кубічні криві Безьє" setup={ this.drawCubic } draw={ this.drawCurve }/>
+</div>
```

Comment:
This looks different from the new content in https://github.com/Pomax/BezierInfo-2/blob/master/docs/chapters/introduction/content.en-GB.md